### ISSUE TITLE
make error

### ISSUE THREAD

**Issue by wuzhi19931128** [NVIDIA/DALI#1654]
Is DALI support for 64-bit ubuntu? 

 I install DALI form source in github and it's really difficult for me. I have to add -DNVJPEG_ROOT_DIR -DJPEG_INCLUDE_DIR -DFFMPEG_ROOT_DIR when I use cmake. IF I don't do that,cmake can work but make is eror. They all have install in system by "make install" BUT I still need to add path /usr/local and so on. I hope there can be batter in later version.

EVEN I have done all the thing which I mentioned,there still a make error:
"/usr/bin/ld : /usr/local/lib/libavcodec.a(vcldsp_MMX.o): relocation R_x86_64_PC32_against symbolic "ff_pw_9" can not be used when making a shared object ; recompile with -fPIC"

What should I do? For convenience, I attach my cmake log and make log here .
[log.txt](https://github.com/NVIDIA/DALI/files/4057751/log.txt)

**Comment by szalpal** [comment]
Hi @wuzhi19931128 ,
How did you install ffmpeg on the machine you're trying to build Dali?

**Comment by JanuszL** [comment]
@wuzhi19931128 - we are aware of those difficulties. As DALI tends to use up to date dependent libraries whenever possible (to have latest optimization and security fixes) while the one delivered with most commonly encountered distributions are usually heavily outdated, it is very hard to make sure the project compiles in all available configurations. That is why we recommend using Docker as a build environment - https://docs.nvidia.com/deeplearning/sdk/dali-developer-guide/docs/compilation.html#building-python-wheel-and-optionally-docker-image.

**Comment by wuzhi19931128** [comment]
I install ffmpeg followed the guide in ffmpeg website, but there are errors in make process. Now , I reinstall ffmpeg using the commod in NVIDA website and it works . This common seems just install part of ffmpeg.

Now ,I already installed dali in my computer but it takes me about two days. In fact, I have tried docker to install dali but there are also some unexpected errors. So maybe it's better to make the peocess easier .

Thanks anyway.
